### ISSUE TITLE
same images in a pair

### ISSUE THREAD

**Issue by midiexianghhh** [yuvalkirstain/PickScore#30]
I looked at some image pairs, both images in a pair are absolutely same. Does it makes sense?

**Comment by yuvalkirstain** [comment]
No, it should not happen. Perhaps write here a minimal example for how to reproduce this issue?
